src/Microsoft.Framework.PackageManager/Packing/NativeImageGenerator.cs

```diff
@@ -126,7 +126,7 @@ public static NativeImageGenerator Create(PackOptions options, PackRoot root, IE
                 if (VersionUtility.IsDesktop(frameworkName))
                 {
                     options.Reports.Information.WriteLine(
-                        "Native image generation is only supported for KLR Core flavors.".Yellow());
+                        "Native image generation is only supported for dotnet core flavors.".Yellow());
```

Comment:
"CoreCLR flavors"? Sounds a bit better I think... or maybe ".NET Core".

IMHO "dotnet" is really only for cases where ".NET" is inappropriate (package IDs, file/folder names, etc.)